docs/lib/project-setup/fragments/android/project-setup/50_provisionBackend.md

```diff
@@ -0,0 +1,22 @@
+### Step 4: Provision the backend with Amplify CLI
+To start provisioning resources in the backend, change directories to your project directory and run `amplify init`:
+
+```bash
+$ cd ~/Developer/MyAmplifyApp/
```

Comment:
Markdown that's `bash` should not have a prompt. I follow [Markdownlint's recommendation](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output) here.

Reply:
I see, cool.  i will remove the $ in the case where there is no output.